REFERENCE.md

```diff
@@ -1914,6 +1914,15 @@ considered stale. Set this to 'absent' to remove it completely.
 
 Default value: `undef`
 
+##### `ttl_status`
```

Comment:
run `rake reference` to re-generate this as the data type in the docs does not match the code.